catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/stage/ParticleEffectsTest.kt

```diff
@@ -110,9 +112,11 @@ class ParticleEffectsTest {
 
     @Test
     fun particleEffectAfterSceneContinuesTest() {
-        script.addBrick(WaitBrick(200))
-        script.addBrick(FadeParticleEffectBrick(FADE_IN))
-        script.addBrick(SceneStartBrick("scene2"))
+        script.apply {
+            addBrick(WaitBrick(200))
+            addBrick(FadeParticleEffectBrick(FADE_IN))
+            addBrick(SceneStartBrick("scene2"))
+        }
         projectManager.currentlyEditedScene = scene2
         onView(ViewMatchers.withId(R.id.button_play)).perform(click())
         onView(withText("Current scene (scene2)")).perform(click())
```

Comment:
I think we should also improve this line here. This `withText()`-Matcher makes the test pretty brittle because now the test is depending too much on whatever's displayed on the UI, just like with the `SceneTransitionBlock` below.
Maybe the `containsString()`-matcher (http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/core/StringContains.html) can be used here to select the option from the dialog where the name of the target-scene "scene2" appears. Can you try this out please?